charts/cf-runtime/templates/monitor/deployment.monitor.yaml

```diff
@@ -40,7 +40,14 @@ spec:
             - name: PORT
               value: "9020"
             - name: API_TOKEN
-              value: {{ .Values.monitor.token }}
```

Comment:
@ziv-codefresh I know I just put an approval on this but wouldn't this potentially break existing deployments that use the `values.monitor.token` ?

Reply:
It would still work, I've introduced a new secret that would be created on-demand at `charts/cf-runtime/templates/monitor/token-secret.monitor.yml`.